client/src/components/Header.css

```diff
@@ -0,0 +1,32 @@
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap');
+
+header {
+    background-color: #ffffff;
+    height: 72px;
+}
+
+a, a:visited {
+    color: #000000;
+    text-decoration: none;
+}
+
+.header-logo_container {
+    display: flex;
+    align-items: flex-end;
+    flex-shrink: 0;
+    gap: 34px;
```

Comment:
Consider using EM as a multiple of the font-size, e.g. 32px = 2em, 72px = 4.5em. This will make things more consistent and the math a bit easier as well.

Reply:
is it bad practice to have some styles in em and some in px? Or is it best to keep to one unit? I have some styles in the css which is still using px like the logo width and height. Thanks